src/functionaltests/java/com/ericsson/ei/utils/FunctionalTestBase.java

```diff
@@ -75,26 +56,6 @@
     @Value("${aggregated.collection.name}")
     private String aggregatedCollectionName;
```

Comment:
These three fields should also be removed. I did not see then previously in my view of pull request.